### PR TITLE
Add in EZTV support to Magnetissimo.

### PR DESCRIPTION
**Pull request by sergiotapia**
Includes @agustif's work, and some final little tweaks.

### CODE REVIEW

**Comment by sergiotapia** [comment]
@agustif Check out what I did here, I'm going to close #10 since I don't know how to push code to a pull-request. 

At this stage the tests work, and the parser works. You just need to invoke it in the crawler, create download workers for EZTV and finally make sure there's a exq queue set up for EZTV. 

Let me know if you need any help. Thanks!

File: lib/parsers/eztv.ex [hunk 1]
```diff
@@ -15,38 +15,43 @@ defmodule Magnetissimo.Parsers.EZTV do
...
       |> Floki.find("td.section_post_header")
+      |> Enum.at(0)
       |> Floki.text
       |> String.trim
 
     magnet = html_body
-      |> Floki.find("td.episode_middle_column a")
+      |> Floki.find("a")
       |> Floki.attribute("href")
       |> Enum.filter(fn(url) -> String.starts_with?(url, "magnet:") end)
```
Comment (sergiotapia) [inline lib/parsers/eztv.ex]: We don't need to filter anything except, "Is this a hyperlink?" - then "Does it start with `magnet:`? and we're all set.

**Comment by agustif** [comment]
I have it working on local. I removed my database and put 0 queue for all the other parsers.

So far 18500 torrents indexed. for some suspciously big numbers I think something may be wrong in the seedes/leechers selector.

https://www.dropbox.com/s/u1obn7wjej8j2hm/Screen%20Shot%202016-09-08%20at%207.46.33%20PM.png?dl=0

**Comment by TehNut** [comment]
Possibly the same (or similar) issue as #8

**Comment by agustif** [comment]
I can't push my code directly to sergiotapia/magnetissimo. should I open a new PR on my own repo that you can later merge with this PR? I dont know...

**Comment by sergiotapia** [comment]
Hey @agustif good stuff! I think you need to open a branch and new PR on your fork and request a pull request, I can then merge it in. Awesome!

**Comment by agustif** [comment]
I think the seeds may be wrong due to enum selection.

Im getting a lot of errors on my log from the Magnetissimo.SizeConverter.size_to_bytes, having passed as arguments 2014 (which almost always appears to be a year) and nil.

Maybe in some files there is less info displayed and relying on enum.at(6) is not reliable for scraping that info, idk.

**Comment by sergiotapia** [comment]
@agustif I moved over your changes directly to master. Thanks for the work you put in, Magnetissimo now has EZTV support. 🙌 

screen shot 2016-09-11 at 8 37 33 am

I'm going to go ahead and close this PR.

**Comment by agustif** [comment]
Thanks @sergiotapia glad to be able to help, I am a designer coming to Elixir/Phoenix and this is really good for me to get real world experience without compromising on jobs yet.

**Pull request closed without merging**